addon/components/flexberry-groupedit.js

```diff
@@ -1,317 +1,416 @@
+    When the component is placed on edit form then this property is initiated as:
+    ```handlebars
+     { {flexberry-groupedit
+       ...
+       sorting=computedSorting)} }
+    ```
+
+    where {{#crossLink "SortableControllerMixin/computedSorting:property"}}computedSorting is property of SortableControllerMixin{{/crossLink}}.
+
+    @property sorting
+    @type Object
+  */
   sorting: null,
 
   /**
-   * Model projection which should be used to display given content.
-   *
-   * @property modelProjection
-   * @type Object
-   * @default null
-   */
-  modelProjection: null,
+    Flag: indicates whether table are striped.
 
-  /**
-   * Content to be displayed (models collection).
-   *
-   * @property content
-   * @type ManyArray
-   * @default null
-   */
-  content: null,
+    @property tableStriped
+    @type Boolean
+    @default true
+  */
+  tableStriped: true,
 
   /**
-   * Flag: indicates whether DELETE request should be immediately sended to server (on each deleted record) or not.
-   *
-   * @property immediateDelete
-   * @type Boolean
-   * @default false
-   */
-  immediateDelete: false,
+    Flag: indicates whether to use single column to display all model properties or not.
 
-  /**
-   * Flag: indicates whether records should be edited on separate route.
-   *
-   * @property editOnSeparateRoute
-   * @type Boolean
-   * @default false
-   */
-  editOnSeparateRoute: false,
+    @property useSingleColumn
+    @type Boolean
+    @default false
+  */
+  useSingleColumn: false,
 
-  /**
-   * Flag: indicates whether to save current model before going to the detail's route.
-   *
-   * @property saveBeforeRouteLeave
-   * @type Boolean
-   * @default false
-   */
-  saveBeforeRouteLeave: false
+  actions: {
+    /**
+      Handles action from object-list-view when no handler for this component is defined.
+
+      @example
+      If sorting is used then there has to be declaration:
+      ```
+      {{flexberry-groupedit
+       ...
+       orderable=true
+       sorting=computedSorting
+       sortByColumn=(action "sortByColumn")
+       addColumnToSorting=(action "addColumnToSorting")}}
+      ```
+
+      @method actions.sortByColumn
+      @param {Object} column Column to sort by.
+    */
+    sortByColumn(column) {
+      throw new Error('No handler for sortByColumn action set for flexberry-groupedit. ' +
+                      'Set handler like {{flexberry-groupedit ... sortByColumn=(action "sortByColumn")}}.');
+    },
+
+    /**
+      Handles action from object-list-view when no handler for this component is defined.
+
+      @example
+      If sorting is used then there has to be declaration:
+      ```
+      {{flexberry-groupedit
+       ...
+       orderable=true
+       sorting=computedSorting
+       sortByColumn=(action "sortByColumn")
+       addColumnToSorting=(action "addColumnToSorting")}}
+      ```
+
+      @method actions.addColumnToSorting
+      @param {Object} column Column to add sorting by.
```

Comment:
To my mind a dot has to be set at the end of the parameter comment. We use uppercase letter at the beginning so we have to set a dot.